### PR TITLE
Increase line length limit 

### PR DESCRIPTION
**Pull request by pirgeo**
Removed the test for line length limit for now since I didn't find a way to test it (it currently is impossible to run into this limit)

### CODE REVIEW

**Review by pichlermarc — APPROVED** [review]
Looks good, I tried meeting the line length limit but gave up at around 17k characters. :slightly_smiling_face:

File: test/Dynatrace.MetricUtils.Tests/DynatraceMetricsSerializerTests.cs [hunk 1]
```diff
@@ -345,24 +345,12 @@ public void TestMetricKeyEmpty()
...
-
-			var serializer = new DynatraceMetricsSerializer(Logger);
-			var metric = DynatraceMetricsFactory.CreateLongGauge("metric", 4, dimensions);
-
-			FluentActions.Invoking(() => serializer.SerializeMetric(metric)).Should().Throw<DynatraceMetricException>()
-				.WithMessage("Metric line exceeds line length of 2000 characters (Metric name: 'metric').");
-		}
+		
+		// [Fact]
+		// public void TestMetricLineTooLong()
```
[the remark this answers is not in the document]
Comment (pichlermarc) [reply test/Dynatrace.MetricUtils.Tests/DynatraceMetricsSerializerTests.cs]: Discussed offline that we will change it to allow more than 50 dimensions, and not check that anymore. This is resolved by [23f22da](https://github.com/dynatrace-oss/dynatrace-metric-utils-dotnet/pull/4/commits/23f22da30d4ea06f4590679eb17cafa23b0d97f4) :slightly_smiling_face:

File: src/Dynatrace.MetricUtils/DynatraceMetricApiConstants.cs [hunk 2]
```diff
@@ -37,6 +39,7 @@ public static class DynatraceMetricApiConstants
 		public static int PayloadLinesLimit => LinesLimit;
 
 		/// <summary>The maximum number of dimensions that can be attached to one metric key.</summary>
+		[Obsolete("Dimensions are no longer dropped.")]
```
Comment (joaopgrassi) [inline src/Dynatrace.MetricUtils/DynatraceMetricApiConstants.cs:42]:
```suggestion
		[Obsolete("All dimensions passed to the serializer are added")]
```

Nit: Maybe something like this instead?

**Review by joaopgrassi — APPROVED** [review]

**Comment by pirgeo** [comment]
I will bump the .NET version in a separate PR

**Pull request merged**